src/status_im/contexts/onboarding/create_password/view.cljs

```diff
@@ -205,12 +198,10 @@
         :passwords-match?          same-passwords?
         :empty-password?           empty-password?
         :show-password-validation? show-password-validation?
-        :on-input-focus            #(set-focused-input :password)
         :on-change-password        (fn [new-value]
                                      (set-password new-value)
                                      (when same-password-length?
                                        (set-show-password-validation? true)))
-
         :on-change-repeat-password (fn [new-value]
```

Comment:
Not coming from your PR, but since you are touching this file. The component `password-inputs` is re-processed by Reagent on every key press because the handlers are not stable. Because we are dealing with inputs, I think it's worth minimizing unnecessary overhead. If you stabilize the functions using `rn/use-callback`, `password-inputs` should be re-processed by Reagent only when `empty-password?` or `same-password-length?` change.

Reply:
Sure!

Actually I discarded some changes because the review would be harder, but it's a good idea to improve this screen 👍

I'll apply your suggestion. Thanks